Paper.tex

```diff
@@ -106,6 +106,7 @@ \subsection{Previous Work} \label{ch:previous}
 Early work on smart contracts has been done by \cite{szabo1997formalizing} and \cite{miller1997future}. Around the 1990s it became clear that algorithmic enforcement of agreements could become a significant force in human cooperation. Though no specific system was proposed to implement such a system, it was proposed that the future of law would be heavily affected by such systems. In this light, Ethereum may be seen as a general implementation of such a \textit{crypto-law} system.
 
 %E language?
+For a list of terms used in this paper, refer to Appendix \ref{ch:Terminology}.
```

Comment:
`ch:Terminology` is not defined anywhere.

Reply:
It is now.